website/django/data/management/commands/add_papers.py

```diff
@@ -27,7 +27,7 @@ def handle(self, *args, **options):
         existing_variant_papers.update(deleted=True)
 
         paper_objects = {}
-        for pmid, paper in papers.iteritems():
+        for pmid, paper in list(papers.items()):
```

Comment:
I assume these changes where a generator is wrapped in a call to `list()` are introduced by `2to3`. It's not wrong per se, but it's a surprisingly conservative choice on their part to convert every generator to a list -- in the cases I've seen later in the file they'd all be fine since they're being iterated over anyway. Anyway, no action needed, but I thought it was strange.